README.md

```diff
@@ -2581,148 +2569,86 @@ After enabling the client statistics, you can monitor your clients using Hazelca
+INFO:hazelcast.connection:Trying to connect to Address(host=127.0.0.1, port=5701)
+INFO:hazelcast.lifecycle:HazelcastClient 4.0.0 is CONNECTED
+INFO:hazelcast.connection:Authenticated with server Address(host=172.17.0.2, port=5701):7682c357-3bec-4841-b330-6f9ae0c08253, server version: 4.0, local address: Address(host=127.0.0.1, port=56758)
+INFO:hazelcast.cluster:
 
-class VersionMessageFilter(logging.Filter):
-    def filter(self, record):
-        record.version_message = "[" + CLIENT_VERSION + "]"
-        return True
-        
-class HazelcastFormatter(logging.Formatter):
-    def format(self, record):
-        client_name = getattr(record, "client_name", None)
-        cluster_name = getattr(record, "cluster_name", None)
-        if client_name and cluster_name:
-            record.msg = "[" + cluster_name + "] [" + client_name + "] " + record.msg
-        return super(HazelcastFormatter, self).format(record)
-```
-
-```python
-logging_config = {
-    "version": 1,
-    "disable_existing_loggers": False,
-    "filters": {
-        "version_message_filter": {
-            "()": "some_package.log.VersionMessageFilter"
-        }
-    },
-    "formatters": {
-        "hazelcast_formatter": {
-            "()": "some_package.log.HazelcastFormatter",
-            "format": "%(asctime)s %(name)s\n%(levelname)s: %(version_message)s %(message)s",
-            "datefmt": "%b %d, %Y %I:%M:%S %p"
-        }
-    },
-    "handlers": {
-        "console_handler": {
-            "class": "logging.StreamHandler",
-            "stream": "ext://sys.stdout",
-            "filters": ["version_message_filter"],
-            "formatter": "hazelcast_formatter"
-        }
-    },
-    "loggers": {
-        "HazelcastClient": {
-            "handlers": ["console_handler"],
-            "level": "INFO"
-        }
-    }
+Members [1] {
+	Member [172.17.0.2]:5701 - 7682c357-3bec-4841-b330-6f9ae0c08253
 }
 
-client = hazelcast.HazelcastClient(
-    logging_config=logging_config
-)
-
-## Some operations
-
-client.shutdown()
-``` 
+INFO:hazelcast.client:Client started
+INFO:hazelcast.lifecycle:HazelcastClient 4.0.0 is SHUTTING_DOWN
+INFO:hazelcast.connection:Removed connection to Address(host=127.0.0.1, port=5701):7682c357-3bec-4841-b330-6f9ae0c08253, connection: Connection(id=0, live=False, remote_address=Address(host=172.17.0.2, port=5701))
+INFO:hazelcast.lifecycle:HazelcastClient 4.0.0 is DISCONNECTED
+INFO:hazelcast.lifecycle:HazelcastClient 4.0.0 is SHUTDOWN
+```
 
-To learn more about the `logging` module and its capabilities, please see the [logging cookbook](https://docs.python.org/3/howto/logging-cookbook.html) and [documentation](https://docs.python.org/3/library/logging.html) of the `logging` module.
+To learn more about the `logging` package and its capabilities, please see the [logging cookbook](https://docs.python.org/3/howto/logging-cookbook.html) and [documentation](https://docs.python.org/3/library/logging.html) of the `logging` package.
```

Comment:
nit: does it make sense to also put links for Python 2 somewhere here, as we support both versions?

Reply:
I don't think it is necessary. Python2 users might switch to their version using the dropdown menu on the navbar in that link.

Reply:
OK, makes sense. Let's leave it as is then.